noggin/templates/user-settings-profile.html

```diff
@@ -12,7 +12,7 @@
     {% set buttontext = _('Change Avatar')%}
     {%- if OPENID is defined %}
     <form method="GET" action="https://www.libravatar.org/openid/login/">
-      <input type="hidden" name="openid_identifier" value="{{ 'https://'+user.username+OPENID }}">
+      <input type="hidden" name="openid_identifier" value="{{ OPENID % format(user.username) }}">
```

Comment:
This crashes Jinja, it's either `OPENID % user.username` or `OPENID|format(user.username)`. The [docs](https://jinja.palletsprojects.com/en/2.11.x/templates/#format) suggest the former.

Reply:
This should be fixed now.